projects/plugins/protect/src/js/routes/scan/scan-results-data-view.tsx

```diff
@@ -18,39 +27,97 @@ export default function ScanResultsDataView( {
 }: {
 	filters: React.ComponentProps< typeof ThreatsDataViews >[ 'filters' ];
 } ) {
-	const { setModal } = useModal();
+	const { siteSuffix, blogID } = window.jetpackProtectInitialState;
+	const queryClient = useQueryClient();
 
+	const { wafSupported } = useWafData();
```

Comment:
Does using `wafSupported` for an environment variable check for displaying a Codeable redirect make sense? We want to avoid showing this to Atomic users.